MachObfuscator/Mach/Parsing/Mach+classNames.swift

```diff
@@ -2,7 +2,8 @@ import Foundation
 
 extension Mach {
     var classNames: [String] {
-        guard let classNameSection = objcClassNameSection
+        guard let classNameSection = objcClassNameSection,
```

Comment:
does it crash when className section is empty?

Reply:
To be honest, I only had an issue with empty export trie, but I decided to apply similar fix in all the similar places. In case of `classNames` it looks like the method uses the whole range , not just the `lowerBound` index – so there would be no crash anyway. Let's think about this as early-optimization ;) Do you think I should remove it?

Reply:
with additional checks code is more consistent, I think they may stay